### DISCUSSION TITLE
Expose uncrustify config

### DISCUSSION THREAD

**Discussion by sonnyp** [Prince781/vala-language-server#301] [Ideas]
Hello,

I'm trying to add Vala CI to Workbench and its demos.
I'd like to use the uncrustify config used by VLS but the config isn't exposed and I don't want to manually keep a sync.

Could you move the uncrustify config to a file that is exposed somewhere and from which vls would read from?

**Comment by lw64** [comment]
I think it would be good to have either a separate executable `vala-format` or parameter of vls that formats (or lints) vala files. That way people don't need to mess around with config files and the uncrustify cli.
